products/sle/main.pm

```diff
@@ -1248,6 +1245,10 @@ elsif (get_var("FILESYSTEM_TEST")) {
     }
     load_filesystem_tests();
 }
+elsif (snapper_is_applicable() && get_var("BOOT_TO_SNAPSHOT")) {
+    loadtest "installation/grub_test";
+    loadtest "installation/boot_into_snapshot";
+}
```

Comment:
In line L621 there was `if (!s390x…)` which is now not executed so I assume this would also be executed on s390x, xen, etc. and fail therefore. You might not be able to run a full test on s390x but you can "simulate" it by setting the according parameters and start the test just to see which modules would be scheduled in the main.pm